test/schema/me/bidders.js

```diff
@@ -1,23 +1,8 @@
-import schema from "schema"
 import { runAuthenticatedQuery } from "test/utils"
+import gravity from "lib/loaders/gravity"
```

Comment:
We're getting the mocked module, which `test/utils` is setting up as a series of jest mocks instead of the originals